CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.8)
+project(boost-assert)
+
+add_library(boost_assert INTERFACE)
+add_library(Boost::assert ALIAS boost_assert)
+
+target_include_directories(boost_assert INTERFACE include)
```

Comment:
Some people use `${CMAKE_CURRENT_SOURCE_DIR}/include` here, others `$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>`. Not sure which is correct, or what the difference is.

Reply:
Word on the street is that the correct form is

```
target_include_directories(
  ${LIBNAME}
  PUBLIC
  $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/include>
  $<INSTALL_INTERFACE:include>
  )
```